pytest_tests/tests/failovers/test_failover_storage.py

```diff
@@ -10,43 +10,33 @@
 from helpers.wellknown_acl import PUBLIC_ACL
 from neofs_env.neofs_env_test_base import NeofsEnvTestBase
 from neofs_testlib.env.env import NeoFSEnv, StorageNode
-from neofs_testlib.shell import CommandOptions
 
 logger = logging.getLogger("NeoLogger")
 stopped_nodes: list[StorageNode] = []
 
 
-@pytest.fixture(scope="function", autouse=True)
+@pytest.fixture
 @allure.step("Return all stopped hosts")
-def after_run_return_all_stopped_hosts(neofs_env: NeoFSEnv):
+def after_run_return_all_stopped_storage_nodes(neofs_env: NeoFSEnv):
     yield
-    return_stopped_hosts(neofs_env)
+    return_stopped_storage_nodes(neofs_env)
 
 
-def panic_reboot_host(neofs_env: NeoFSEnv) -> None:
-    shell = neofs_env.shell
-    shell.exec('sudo sh -c "echo 1 > /proc/sys/kernel/sysrq"')
-
-    options = CommandOptions(close_stdin=True, timeout=1, check=False)
-    shell.exec('sudo sh -c "echo b > /proc/sysrq-trigger"', options)
-
-
-def return_stopped_hosts(neofs_env: NeoFSEnv) -> None:
+def return_stopped_storage_nodes(neofs_env: NeoFSEnv) -> None:
     for node in list(stopped_nodes):
-        with allure.step(f"Start host {node}"):
-            node.host.start_host()
+        with allure.step(f"Start {node}"):
+            node.start(fresh=False)
         stopped_nodes.remove(node)
 
     wait_all_storage_nodes_returned(neofs_env)
 
 
 @pytest.mark.failover
-@pytest.mark.skip(reason="These tests require multiple hosts to run")
 class TestFailoverStorage(NeofsEnvTestBase):
-    @allure.title("Lose and return storage node's host")
-    @pytest.mark.parametrize("hard_reboot", [True, False])
```

Comment:
You can still have hard/soft. IIRC you're TERMinating process normally, but you can also KILL it.

Reply:
Added kill case as well.